modules/tile-converter/docs/cli-reference/tile-converter.md

```diff
@@ -38,3 +42,17 @@ $ npx tile-converter --help
 | token                | \*                        |                           | Token for Cesium ION tilesets authentication                                                                                                                                                                                                         |
 | no-draco             | \*                        |                           | Disable draco compression for geometry. Default: not set                                                                                                                                                                                             |
 | help                 | \*                        | \*                        | Show the converter tool options list                                                                                                                                                                                                                 |
+
+## Running local server to handle i3s layer.
+
+After conversion there are new i3s layers in output ("data" in example) directory.
+
+Run it with the local web server from project directory:
+
+```bash
+$ I3sLayerPath="./data/CairoLayer" DEBUG=i3s-server:* npx i3s-server
```

Comment:
Can we make this into arguments on the i3s-server so we don't need to set environment variables? Separate PR of course.

Reply:
Ok, Need to refactor it in another PR. Thanks!